Document experimental status of odr

Add documentation of the experimental status of the odr optimization in dwz.1.

2021-02-25  Tom de Vries  <tdevries@suse.de>

	PR dwz/27401
	* dwz.1: Document experimental status of odr.

---
 dwz.1 | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/dwz.1 b/dwz.1
index 32df953..63c50d5 100644
--- a/dwz.1
+++ b/dwz.1
@@ -112,11 +112,13 @@ corresponding forms, instead of the GNU extension \fI.gnu_debugaltlink\fR
 and corresponding forms.
 .TP
 .B \-\-odr / \-\-no-odr
+.B Experimental.
 Enable/disable One-Definition-Rule optimization for C++ compilation units.
 This optimization causes struct/union/class DIEs with the same name to be
 considered equal.  This has the effect that DIEs referring to distinct DIEs
 representing the same type (like f.i. pointer type DIEs) are considered equal,
-and may be deduplicated.
+and may be deduplicated.  The status of this optimization is experimental.
+It's disabled in low-mem mode.
 Enabled by default.
 .TP
 .B \-\-odr-mode=<basic|link>
